vidos-config-cypress.json

```
@@ -6,6 +6,7 @@
     ],
     "API_SEARCH": "https://search-dev.teritorio.xyz/search",
     "API_SEARCH_ADDR": "https://api-adresse.data.gouv.fr/search",
+    "API_REVERSE_ADDR": "https://api-adresse.data.gouv.fr/reverse",
```

Should be refactored as 
```suggestion
    "API_ADDR": "https://api-adresse.data.gouv.fr",
```